packages/brick_offline_first/lib/src/offline_queue/request_graphql_sqlite_cache_manager.dart

```diff
@@ -0,0 +1,89 @@
+// ignore_for_file: constant_identifier_names
+
+import 'dart:convert';
+import 'package:brick_offline_first/src/offline_queue/request_sqlite_cache_manager.dart';
+import 'package:graphql/client.dart';
+
+class RequestGraphqlSqliteCacheManager extends RequestSqliteCacheManager<Request> {
+  RequestGraphqlSqliteCacheManager(String databaseName)
+      : super(databaseName,
+            tableName: GRAPHQL_JOB_TABLE_NAME,
+            createdAtColumn: GRAPHQL_JOB_CREATED_AT_COLUMN,
+            primaryKeyColumn: GRAPHQL_JOB_PRIMARY_KEY_COLUMN,
+            lockedColumn: GRAPHQL_JOB_LOCKED_COLUMN,
+            updateAtColumn: GRAPHQL_JOB_UPDATED_AT);
```

Comment:
also these should be alphabetized
```suggestion
            updateAtColumn: GRAPHQL_JOB_UPDATED_AT,);
```